.github/workflows/test-opa-coverage.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
       - name: OPA test coverage
         run: |
-          coverage=$( ./opa test ./bundle -c | jq '.coverage' )
+          coverage=$(opa test ./bundle -c | jq '.coverage' )
```

Comment:
Can we also trim the precision?
![image](https://github.com/elastic/cloudbeat/assets/85433724/089bb2c9-8c15-4b75-a541-fa6cace84281)